### DISCUSSION TITLE
Embedded Text Editor + Tailwind CSS

### DISCUSSION THREAD

**Discussion by myregistration** [tailwindlabs/tailwindcss#10068] [Help]
I integrated Tailwind CSS into our website and love it, so now I want to create a UI with an embedded text editor for our non-techie team members to update page content frequently, like a CMS.  I wasn't able to use TINYMCE or Trix because they don't parse the Tailwind styles.  And although I could update source code with one of the embedded editors, it still didn't parse the styles when I viewed the HTML.   Anyone know of an editor I can integrate that supports Tailwind CSS?  Thanks!

**Comment by wongjn** [comment]
> I wasn't able to use TINYMCE or Trix because they don't parse the Tailwind styles.

How are you trying to get these editors to "parse the Tailwind styles"?

> And although I could update source code with one of the embedded editors, it still didn't parse the styles when I viewed the HTML.

Again, not quite sure what you mean by "parse the styles when I viewed the HTML" here.

Without quite understanding how you are trying to integrate the CSS generated by Tailwind into the editor, you could instead look at using the [`@tailwindcss/typography` plugin](https://tailwindcss.com/docs/typography-plugin) which <q>you can use to add beautiful typographic defaults to any vanilla HTML you don’t control, like HTML rendered from Markdown, or pulled from a CMS.</q>

**Reply by myregistration** [reply]
I'm basically trying to create a UI that acts like a CMS to allow the team to update content that displays in webpages.  When I say parsing I am referring to wanting to be able to view both Source and HTML so I can see the changes before saving them.  I was able to do this with Livewire and AlpineJS (I'm using Laravel framework) via a textarea field for input and a div for displaying changes realtime.  It was hoping there was a text editor I could embed in the web page that could Preview that HTML and Source code and worked with Tailwind CSS tags.  And when I edit the Source with TinyMCE and Save it strips the classes out.  I'll do some more research.
